### PR TITLE
Update the formatting to display answers better

### PR DESCRIPTION
**Pull request by injms**
Contact details, date of birth, and address all needed special consideration to make them more understandable.

## Desktop

image

## Mobile

image

### CODE REVIEW

File: app/controllers/coronavirus_form/check_answers_controller.rb [hunk 1]
```diff
@@ -36,20 +36,38 @@ def items
...
     end
   end
 
-  def concat_answer(answer)
-    answer.values.compact.join(" ")
+  def concat_answer(answer, question)
+    concatenated_answer = []
+    joiner = " "
+
+    if question.eql?("contact_details")
```
Comment (injms) [inline app/controllers/coronavirus_form/check_answers_controller.rb:55]: I realise that the following is quite write-everything-twice, but I think it's a touch clearer for future people to see what we're doing here.

File: app/controllers/coronavirus_form/check_answers_controller.rb [hunk 2]
```diff
@@ -36,20 +36,38 @@ def items
...
+      concatenated_answer << "Phone number: #{answer['phone_number_calls']}" unless answer["phone_number_calls"].nil?
```
Comment (leenagupte) [inline app/controllers/coronavirus_form/check_answers_controller.rb]: You changed the `unless` and drop the `nil`
e.g.
```
concatenated_answer << "Phone number: #{answer['phone_number_calls']}" if answer["phone_number_calls"]
```

Comment (injms) [reply app/controllers/coronavirus_form/check_answers_controller.rb]: Much better - thank you!

**Review by leenagupte — APPROVED** [review]
Looks good! There might be a neater way of doing this, but it's late in day, so go for it! 👍

**Pull request merged**